### ISSUE TITLE
The link to the Ohio State home page in the banner displays “OSU.EDU” but a screen reader announces it as “The Ohio State University”

### ISSUE THREAD

**Issue by McManning** [osuresearch/ui#87]
The link to the Ohio State home page in the banner displays “OSU.EDU” but a screen reader announces it as “The Ohio State University”. Logos especially should announce exactly as how they are read if made of text. 
 
•	[WCAG 2.5.3 Label in Name](https://www.w3.org/WAI/WCAG21/quickref/#label-in-name)
•	Recommendation: Since it is a background image, remove the link text as it both inaccurate and redundant. Change the title attribute to read, “OSU.EDU, home”. 
•	Priority: Moderate

**Comment by helloLilyX** [comment]
Do I ignore `Anchors must have content and the content must be accessible by a screen reader.eslint[jsx-a11y/anchor-has-content](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/tree/HEAD/docs/rules/anchor-has-content.md)`?

**Comment by McManning** [comment]
I think let's try swapping it around, instead of using title let's just have the anchor text be "OSU.EDU, home". That should be read off by SR's, even if it's visibility hidden from sighted users.
